packages/webapp/src/components/Modals/QuickAssignModal/index.jsx

```diff
@@ -48,6 +65,11 @@ export default function TaskQuickAssignModal({
     dismissModal();
   };
 
+  const onCheckedAll = () => {
+    checkUnassignedTaskForSameDate();
```

Comment:
Is there a reason for this statement? The return value is not used, and I don't see a side effect from the call.

Reply:
I have removed that, I was calling that function on the checkbox click, to test out the function's results.